content/en/docs/guides/backup-and-restore.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Back up and Restore
+linkTitle: Back up and Restore
+description: "How to back up and restore resources in cozystack cluster."
+weight: 40
+aliases:
+  - /docs/backup-and-restore
+---
+
+CozyStack uses [Velero](https://velero.io/docs/v1.16/) to manage Kubernetes resource backups and recovery, including volume snapshots.
+This guide explains how to configure Velero and perform backups and recovery with practical examples.
+
+
+## Prerequisites
+
+To back up data in Cozystack, you need an S3-compatible storage ready.
+
+We recommend using the Velero CLI: [https://velero.io/docs/v1.16/basic-install/#install-the-cli](https://velero.io/docs/v1.16/basic-install/#install-the-cli)
+
+## 1. Set up Storage Credentials and Configuration
+
+To enable backups, the first step is to provide Cozystack with access to an S3-compatible storage.
+
+All configuration settings are created as secrets in the `cozy-velero` namespace.
+
+### 1.1 Create a Secret with S3 Credentials
+
+Create a secret containing credentials for your S3-compatible storage where backups will be saved.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-credentials
+  namespace: cozy-velero
+type: Opaque
+stringData:
+  cloud: |
+    [default]
+    aws_access_key_id=<KEY>
+    aws_secret_access_key=<SECRET KEY>
+
+    services = seaweed-s3
+    [services seaweed-s3]
+    s3 =
+        endpoint_url = https://s3.tenant-name.cozystack.example.com
+```
```

Comment:
After applying, how do I check that credentials are fine and Cozystack/Velero has access to the bucket?